tests/unit/controllers/signup-test.js

```diff
@@ -5,23 +5,12 @@ module('Unit | Controller | signup', function (hooks) {
   setupTest(hooks);
   test('it test', function (assert) {
     let controller = this.owner.lookup('controller:signup');
-    assert.expect(46);
+    assert.expect(43);
     assert.equal(
       controller.isSubmitClicked,
```

Comment:
43 asserts in one test seems like an overkill. Also, 'it test' doesn't tell me what this test is doing.

Reply:
We do have - `it test`
The assertions were reduced because we moved the new-sign up logic to a different controller and this test file had all the tests for the new sign-up, and this was causing the tests to fail.